.gitignore

```diff
@@ -41,6 +41,9 @@ output/*/index.html
 docs/_build
 docs/modules.rst
 
+# LevelDB
+evm/db/backends/db
```

Comment:
@pipermerriam So you're saying the db files should be stored outside of `./evm` (the only thing which should be in `evm/db/backends/db` should be db files that are constantly being created and deleted in the `__delete__` method of `LevelDB`)?  Should the db files db created in a `db/` folder at the root level of the repo?